src/sqlfluff/rules/L011.py

```diff
@@ -68,14 +68,14 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
 
                             # Remove the AS as we're using implict aliasing
                             fixes.append(LintFix.delete(context.segment.segments[0]))
-                            anchor = context.raw_stack[-1]
```

Comment:
Switching from `raw_stack` to `raw_segment_pre` so this rule doesn't require the heavyweight `raw_stack` anymore.